docs/src/for-developers/variational_inference.md

```diff
@@ -20,257 +20,257 @@ In Bayesian inference one usually specifies a model as follows: given data ``$\{
 \end{align*}
 ```
 
-where ``$\overset{\text{i.i.d.}}{\sim}$`` denotes that the samples are identically independently distributed. Our goal in Bayesian inference is then to find the _posterior_
+where ``\overset{\text{i.i.d.}}{\sim}`` denotes that the samples are identically independently distributed. Our goal in Bayesian inference is then to find the _posterior_
 
 ```math
-p(z \mid \{ x_i \}_{i = 1}^n) = \prod_{i=1}^{n} p(z \mid x_i).
+p(z \mid \\{ x\_i \\}\_{i = 1}^n) = \prod\_{i=1}^{n} p(z \mid x\_i).
 ```
 
-In general one cannot obtain a closed form expression for <span>``$p(z \mid \{ x_i \}_{i = 1}^n)$``</span>, but one might still be able to _sample_ from ``$p(z \mid \{ x_i \}_{i = 1}^n)$`` with guarantees of converging to the target posterior ``$p(z \mid \{ x_i \}_{i = 1}^n)$`` as the number of samples go to ``$\infty$``, e.g. MCMC.
+In general one cannot obtain a closed form expression for <span>``p(z \mid \\{ x\_i \\}\_{i = 1}^n)``</span>, but one might still be able to _sample_ from ``p(z \mid \\{ x\_i \\}\_{i = 1}^n)`` with guarantees of converging to the target posterior ``p(z \mid \\{ x\_i \\}\_{i = 1}^n)`` as the number of samples go to ``\infty``, e.g. MCMC.
```

Comment:
I think `<span>` is another workaround for the underscore issue and could be removed now:
```suggestion
In general one cannot obtain a closed form expression for ``p(z \mid \\{ x\_i \\}\_{i = 1}^n)``, but one might still be able to _sample_ from ``p(z \mid \\{ x\_i \\}\_{i = 1}^n)`` with guarantees of converging to the target posterior ``p(z \mid \\{ x\_i \\}\_{i = 1}^n)`` as the number of samples go to ``\infty``, e.g. MCMC.
```